### ISSUE TITLE
NTR, pulmo: Accumulation of neuroendocrine cells in the bronchioles

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6390]
**Preferred term label:**Accumulation of neuroendocrine cells in the bronchioles

**Synonyms** Hyperplasia of neuroendocrine cells, 
Neuroendocrine Cell Hyperplasia of Infancy (NEHI)

**Definition (free text, please give PubMed ID)**
Presence of increased numbers of bombesin-immuno-positive neuroendocrine cells within distal airways 

**Parent term (use hpo.jax.org/app)**
Interstitial pulmonary abnormality HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:217560 Neuroendocrine Cell Hyperplasia of Infancy (NEHI)

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by GrieseAG** [comment]
Comment: There are no formal criteria for NEHI diagnosis, based on excess of neuroendocrine cells (NECs). But it has been suggested that findings of NECs within >= 70% of bronchioles in the lung biopsy specimen and >= 10% NECs in an individual airway are consistent with the
diagnosis in the appropriate clinical setting.

CHEST 2011; 139(5):1060–1071

**Comment by pnrobinson** [comment]
Proposed label
Increased bronchiolar neuroendocrine cells
Def:Presence of increased numbers of bombesin-immuno-positive neuroendocrine cells within distal airways. Although there are no formal criteria, findings of neuroendocrine cells in at least 70% of bronchioles by lung biopsy and at least 10% NECs in an individual airway are consistent with the diagnosis in the appropriate clinical setting.PMID: 20884725

Pulmonary neuroendocrine cells (NECs), which produce bioactive products, including bombesin-like peptide and serotonin, are specialized epithelial cells scattered throughout the conducting airways and as innervated clusters (neuroepithelial bodies [NEBs]). In the fetus, NECs are most abundant in the distal airways where they promote branching morphogenesis, epithelial and
mesenchymal cell proliferation, and surfactant secretion. Postnatally, NECs function as oxygen chemosensors and degranulate in response to hypoxia. Although NECs decline rapidly in number after the neonatal period, NEC hyperplasia has been described in
a number of conditions or disorders, including bronchopulmonary dysplasia, sudden infant death syndrome, pulmonary hypertension, cystic fibrosis, and mechanical ventilation.PMID: 20884725

**Comment by pnrobinson** [comment]
Preferred label -- Increased airway neuroendocrine cells

**Comment by mgriese** [comment]
We have to differentiate a) the pathological finding (qualitative/quantitative) and b) diseases (PTI, NEHI, Tumors). 

Proposed label
Increased airway neuroendocrine cells (instead of bronchiolar)

NORMAL:
"The bronchial mucosa also contains a small cluster of neuroendocrine cells, also known as Kulchitsky cells. They have neurosecretory type granules and can secrete several factors. This includes catecholamine and polypeptide hormones, such as serotonin, calcitonin, and gastrin-releasing factors (bombesin). Like brush cells, these neuroendocrine cells make up only a small portion of mucosal epithelium, around 3%. PMID: 30521210"


Def: Presence of increased numbers of bombesin-immuno-positive neuroendocrine cells within distal airways. There are no formal criteria for an increase (which is also dependend on the sensitivity of the staining method), findings of neuroendocrine cells in at least 70% of bronchioles by lung biopsy and at least 10% NECs in an individual airway are consistent with the diagnosis of Neuroendocrine cell hyperplasia of infancy in the appropriate clinical setting. PMID: 26474448, 20884725. Increases are also seen in other clicnial settings. PMID: 32558323. Neuroendocrine neoplasms of the lung encompass NE tumors (NETs), which split into typical and atypical carcinoids, and NE carcinomas (NECs). PMID 33306420

Pulmonary neuroendocrine cells (NECs), which produce bioactive products, including bombesin-like peptide and serotonin, are specialized epithelial cells scattered throughout the conducting airways and as innervated clusters (neuroepithelial bodies [NEBs]). In the fetus, NECs are most abundant in the distal airways where they promote branching morphogenesis, epithelial and
mesenchymal cell proliferation, and surfactant secretion. Postnatally, NECs function as oxygen chemosensors and degranulate in response to hypoxia. Although NECs decline rapidly in number after the neonatal period, NEC hyperplasia has been described in
a number of conditions or disorders, including bronchopulmonary dysplasia, sudden infant death syndrome, pulmonary hypertension, cystic fibrosis, and mechanical ventilation.PMID: 20884725

**Comment by pnrobinson** [comment]
parent probably https://hpo.jax.org/app/browse/term/HP:0025426

**Comment by pnrobinson** [comment]
added, thanks!
